src/tokenize_sentence/mod.rs

```diff
@@ -79,3 +103,195 @@ impl Tokenizer for UnicodeSentenceTokenizer {
         Box::new(text.split_sentence_bounds())
     }
 }
+
+/// Punctuation sentence tokenizer
+///
+/// This simple tokenizer uses punctuation (default ".", "?", "!") to determine sentence boundaries.
+/// Trailing whitespace is also captured in the preceding sentence.
+///
+/// # Arguments (PunctuationTokenizerParams)
+///
+/// * `punctuation` - Punctuation tokens used to determine boundaries. Only the first "character" using the `chars` method is used.
```

Comment:
I originally had it as a `Vec<char>` but then PyO3 attribute failed:

```
#[cfg_attr(feature = "python", derive(FromPyObject, IntoPyObject))]
```

Presumably because PyO3 hasn't implemented those traits for a `Vec<char>` which is need when converting to a Python dictionary.

So I could either implement those traits or/and make the documentation clearer?